src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -169,22 +174,28 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
                 && (latestStableVersion.version == null || latestPrerelease.version > latestStableVersion.version) &&
                 !latestPrerelease.version.Equals(installedVersion))
             {
-                _versions.Add(new DisplayVersion(latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
+                VersionRange latestPrereleaseVersionRange = VersionRange.Parse(latestPrerelease.version.ToString(), allowFloating: false);
```

Comment:
[`VersionRange` has constructors that takes `NuGetVersion`](https://github.com/NuGet/NuGet.Client/blob/bc11fd124fd87b55532742db2a56f4a324288bcb/src/NuGet.Core/NuGet.Versioning/VersionRange.cs#L23-L67), so it will be much more efficient than converting to a string, then having VersionRange have to parse the string.

Reply:
I tried to do this change, but after looking at [`DisplayVersion` _toString](https://github.com/NuGet/NuGet.Client/blob/bc11fd124fd87b55532742db2a56f4a324288bcb/src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs#L56-L71). I need the OriginalString property that is populated with `VersionRange.Parse`

Reply:
Should the VersionRange constructor set original string when initialized with minVersion only? 
Or do we think that might break things?